Osiris/GUI.cpp

```diff
@@ -924,6 +924,82 @@ void GUI::renderVisualsWindow(bool contentOnly) noexcept
     ImGui::Checkbox("No grass", &config->visuals.noGrass);
     ImGui::Checkbox("No shadows", &config->visuals.noShadows);
     ImGui::Checkbox("Wireframe smoke", &config->visuals.wireframeSmoke);
+    ImGui::Checkbox("Viewmodel XYZ", &config->visuals.viewmodelXYZ.enabled);
+    ImGui::SameLine();
+    bool viewmodelPopup = ImGui::Button("Edit");
```

Comment:
mb better
```cpp
if (ImGui::Button("Edit")
    ImGui::OpenPopup("##viewmodelPopup");
```

Reply:
pushed an update addressing this issue, should look better now.